test/web-auth/cross-origin-authentication.test.js

```diff
@@ -300,8 +304,12 @@ describe('auth0.WebAuth.crossOriginAuthentication', function() {
             },
             function(err) {
               expect(err).to.be.eql({
-                error: 'request_error',
-                error_description: '{"some":"error"}'
+                original: {
+                  error: 'request_error',
+                  error_description: '{"some":"error"}'

```

Comment:
are you sure this particular error description is an object?

Reply:
it's a string, not an object.

Reply:
https://github.com/auth0/auth0.js/blob/master/src/web-auth/cross-origin-authentication.js#L71